index.html

```diff
@@ -124,5 +124,41 @@ <h3>CANOPY</h3>
         </div>
       </div>
     </section>
+
+  <!-- About Myself Section -->
+    <section class="about-myself">
+      <h2>About Myself</h2>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <a class="connect">LET'S CONNECT</a>
+      <ul class="social-links">
+        <li><a href="https://twitter.com/DavidLeiva111"><img src="img/twitter-icon.png" alt="Twitter icon" class="icon"/></a></li>
+        <li><a href="https://linkedin.com/in/1259221-d-leiva"><img src="img/linkedin-icon.png" alt="LinkedIn icon" class="icon"/></a></li>
+        <li><a><img src="img/medium-icon.png" alt="Medium icon" class="icon"/></a></li>
+        <li><a href="https://github.com/MasterLeiva"><img src="img/github-icon.png" alt="GitHub icon" class="icon"/></a></li>
+        <li><a><img src="img/angelist-icon.png" alt="Angelist icon" class="icon"/></a></li>
+      </ul>
+      <button class="main-buttons">Get my resume</button>
```

Comment:
To make your design complete and a replica of the figma template, consider styling the various interactive states for the buttons both in the work and this section according to the below screenshot.

![yyy](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/84930355/4b11934d-f7a5-4efe-a1ff-d272367c9b63)